### DISCUSSION TITLE
View ToolTips in to table

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1380] [Q&A] [answered]
Hello everyone,
my client asked me to insert tooltips on the passage of the mouse over the cells of the "<Table>" component the cells are already managed by "cellRenderer" , for the insertion of the icons, I tried to intercept the passage of the mouse
return <DefaultCell {...props} status={props.cellProps.row.original.status_componente} onMouseOver={onViewClick}, which the passage is intercepted, the const onViewClik where inside there is only html code seems not work
I also tried wrapping the DefaultCell with the tooltips control, but that too failed

I hope you can help me

Thanks for the precious help

**Comment by MichelleJSAucier** [comment, marked as answer]
https://github.com/search?q=repo%3AiTwin%2FiTwinUI+onMouse&type=code

the above search might help.
